### ISSUE TITLE
Be able to choose the viewport to ScrollTo 

### ISSUE THREAD

**Issue by MartinSStewart** [WhileTruu/elm-smooth-scroll#1]
Currently only the body can be used with ScrollTo. Could ScrollTo instead accept a String parameter representing a DOM node id that should be scrolled instead?

**Comment by WhileTruu** [comment]
Hey! Do I understand correctly that you have a scrollable element inside the body in which you want to scroll to something?

**Comment by MartinSStewart** [comment]
Correct. I was to scroll to a given position (in pixels) in a scrollable element.

**Comment by WhileTruu** [comment]
Ok, thanks :)

**Comment by MartinSStewart** [comment]
Earlier I made a modified copy of ScrollTo for my use case. You can use it as reference if you want.
[SmoothScroll.txt](https://github.com/WhileTruu/elm-smooth-scroll/files/3594525/SmoothScroll.txt)
